Resources/Dockerfiles/neo4jescodb.yaml

```diff
@@ -40,9 +44,16 @@ spec:
             cpu: 1500m
             memory: "16G"
         volumeMounts:
+        - name: certificates
```

Comment:
You could have also loaded the PEM files into a kubernetes secret, and used a volume mount to mount them from the secrets... then you wouldn't have them lying around on file shares:

https://kubernetes.io/docs/tasks/inject-data-application/distribute-credentials-secure/

Reply:
The above is more a "fyi,  this is probably worth learning about for in future",  especially if we start using clusters with more things on them

Reply:
Agreed, looks like too much work to squeeze into the end of this sprint